code/game/objects/structures/aliens.dm

```diff
@@ -127,15 +127,20 @@
 	var/last_expand = 0 //last world.time this weed expanded
 	var/growth_cooldown_low = 150
 	var/growth_cooldown_high = 200
-	var/static/list/blacklisted_turfs = typecacheof(list(
-	/turf/open/space,
-	/turf/open/chasm,
-	/turf/open/floor/plating/lava))
+	var/static/list/blacklisted_turfs
 
-/obj/structure/alien/weeds/New()
+/obj/structure/alien/weeds/Initialize()
 	pixel_x = -4
 	pixel_y = -4 //so the sprites line up right in the map editor
 	..()
+
+	if(!blacklisted_turfs)
```

Comment:
But I saw other things do it, something about... lights on New, rather than putting them in Initialize?